src/routes/transactions.rs

```diff
@@ -22,5 +22,5 @@ pub fn details(context: Context, details_id: String) -> Result<content::Json<Str
 
 #[get("/about")]
 pub fn about(context: Context) -> Result<content::Json<String>> {
-    context.cache().cache_resp(&context.uri(), 60 * 200, about::get_about)
+    context.cache().cache_resp(&context.uri(), request_cache_duration(), about::get_about)
```

Comment:
I would use a different cache duration here (as this can theoretically be cached until we deploy a new service). So I would introduce a new one that defaults to a reasonable value (maybe even to the `request_cache_duration()`)

Reply:
Do you mean, we should have an additional env variable `ABOUT_CACHE_DURATION` that defines that?

Reply:
would be fine for me